### PR TITLE
util: fix err check

### PR DESCRIPTION
**Pull request by Tjianke**
### What problem does this PR solve?

Issue Number: #22979  

Problem Summary:

### What is changed and how it works?

What's Changed: 
- added basic err check
- some assert failed so I commented them with `// to fix:`. Needs further action, either ignore these error or fix tests.

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

- No release note

### CODE REVIEW

File: util/chunk/chunk_test.go [hunk 1]
```diff
@@ -631,22 +631,40 @@ func (s *testChunkSuite) TestSwapColumn(c *check.C) {
 		c.Assert(chk2.columns[0] == chk2.columns[1], check.IsTrue)
 	}
 
-	chk1.SwapColumn(0, chk2, 0)
+	err := chk1.SwapColumn(0, chk2, 0)
+	if err != nil {
+		panic(err)
```
Comment (zimulala) [inline util/chunk/chunk_test.go:636]: Why not use `c.Assert`?

Comment (Tjianke) [reply util/chunk/chunk_test.go]: Changed.

**Review by zz-jason — COMMENTED** [review]
LGTM

**Pull request merged**